app.rb

```diff
@@ -0,0 +1,95 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    puts 'List of Books:'
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    puts 'List of People:'
+    @people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}"
+    end
+  end
+
+  def create_person
+    puts 'Enter person type (student or teacher):'
+    person_type = gets.chomp.downcase
+    puts 'Enter person name:'
+    name = gets.chomp
+    puts 'Enter person age:'
+    age = gets.chomp.to_i
+
+    if person_type == 'student'
+      puts 'Enter classroom label:'
+      classroom_label = gets.chomp
+      student = Student.new(@people.length + 1, name, age, classroom: classroom_label)
```

Comment:
- [ ] Good job so far but when I tried to create a new person I am getting these errors on the screenshot below ⬇️  Please make sure to fix the errors and make your app works as expected 👍 
![error](https://github.com/ridabensalem/School-library/assets/6615394/83dd30d4-8505-4a98-a2c3-55513fd8330d)